models/event.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const Build = require('./build');
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Pipeline = require('./pipeline');
+const Scm = require('../core/scm');
+const Workflow = require('../config/workflow');
+
+const buildId = Joi.reach(Build.base, 'id');
+
+const MODEL = {
+    id: Joi
+        .string().hex().length(40)
+        .description('Identifier of this build')
+        .example('4b8d9b530d2e5e297b4f470d5b0a6e1310d29c5e'),
+    builds: Joi
```

Comment:
> Like we don't have a list of builds inside a job. 

The number of builds in an event is static when it's created, whereas the number of builds inside a job can vary. 

That being said, this is more of a premature optimization based on what the UI would find convenient to consume. We should drop the `builds` field here and add it in the future as a backwards-compatible change. 

Regardless of what we do with the `builds` field, the build model will still need to reference the event it's associated with.